modules/log6x/cluster-logging-collector-log-forward-secret-cloudwatch.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-external.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cluster-logging-collector-log-forward-secret-cloudwatch_{context}"]
+= Creating a secret for AWS CloudWatch with an existing AWS role
+If you have an existing role for AWS, you can create a secret for AWS with STS using the `oc create secret --from-literal` command.
+
+.Procedure
+
+* In the CLI, enter the following to generate a secret for AWS:
++
+[source,terminal]
+----
+$ oc create secret generic cw-sts-secret -n openshift-logging --from-literal=role_arn=arn:aws:iam::123456789012:role/my-role_with-permissions
+----
++
+.Example Secret
```

Comment:
Shall we merge this section with  https://github.com/openshift/openshift-docs/pull/88847/files#diff-d5cc864988688f4ce55a41aaa83849b903701285f30ea82add150dd0e3693561